src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -43,6 +43,12 @@ def load_arguments(self, _):
         validator=validate_feature_query_fields,
         arg_type=get_enum_type(['name', 'key', 'label', 'locked', 'last_modified', 'state', 'description', 'conditions'])
     )
+    snapshot_fields_arg_type = CLIArgumentType(
+        nargs='+',
+        help='Customize output fields for Snapshots',
+        validator=validate_snapshot_query_fields,
+        arg_type=get_enum_type(['name', 'etag', 'retention_period', 'filters', 'status', 'created', 'expires', 'size', 'items_count', 'size', 'items_link'])
```

Comment:
status_code missing.

Reply:
Thanks for pointing this out. This has been added.